dom.go

```diff
@@ -245,22 +247,69 @@ func (h *HTML) reconcile(prev *HTML) {
 		h.node.Set("innerHTML", h.innerHTML)
 	}
 
+	h.reconcileChildren(prev, nil)
+}
+
+// reconcileChildren reconciles children of the current HTML against a previous
+// render's DOM nodes.
+func (h *HTML) reconcileChildren(prev *HTML, insertBefore *HTML) {
+	if prev == nil {
+		prev = &HTML{}
+	}
```

Comment:
What does this do?

Reply:
Allows calling `reconcileChildren` on the first pass where the prev render is `nil`, sames as early in `*HTML.reconcile()`.

Reply:
This may actually be unnecessary now, as some of the calling code has changed, will review.